sources/PTYTextView.m

```diff
@@ -5875,7 +5875,8 @@ - (void)mouseHandlerDidInferScrollingIntent:(PTYMouseHandler *)handler trying:(B
 
 - (void)mouseHandlerOpenTargetWithEvent:(NSEvent *)event
                            inBackground:(BOOL)inBackground {
-    if ([self showCommandInfoForEvent:event]) {
+    if ([iTermAdvancedSettingsModel enableCmdClickPromptForShowCommandInfo]
+        && [self showCommandInfoForEvent:event]) {
```

Comment:
I'm not sure this is quite the right place for this - maybe in `showCommandInfoForEvent` would be cleaner - but without something here, the click goes to the new feature even though the fix in `sources/iTermURLActionFactory.m` below results in the UX showing the Smart Selection as the highlight.

Reply:
I can't recall why this is here, but this is the right place for it because you don't want to disable showing the command info when you right click on the offscreen command line view.